### PR TITLE
Create articles-of-association.es.md

### PR DESCRIPTION
**Pull request by AlbaRoza**
Spanish translation of the association articles. Not an official translation.

-----
[View rendered organization/articles-of-association.es.md](https://github.com/publiccodenet/about/blob/AlbaRoza-patch-2/organization/articles-of-association.es.md)

### CODE REVIEW

File: organization/articles-of-association.es.md [hunk 1]
```diff
@@ -0,0 +1,459 @@
...
+
+11.2. Además de los miembros, tendrán acceso a la reunión los directores, los miembros de la y los admitidos por el presidente de la reunión. Todos ellos cuentan con derecho de hablar en la reunión.
+
+11.3. Los miembros también pueden participar en las deliberaciones de la asamblea general a través de un medio de comunicación electrónico. La Junta podrá imponer condiciones para el uso de los medios electrónicos de comunicación. Estas condiciones pueden referirse a la participación en las deliberaciones y a la emisión de votos. La junta anunciará cualquier condición establecida en el aviso de convocatoria.
+
+11.4. El presidente de la junta también actuará como tal en las reuniones de la junta general de accionistas. Si el presidente de la junta no está presente, la junta se encargará de su propia gestión.
+
+11.5. El secretario de la junta directiva o, en su defecto, la persona designada por el presidente, levantará acta de las deliberaciones de la reunión. Estas actas se aprobarán en la misma o en la siguiente reunión y serán firmadas por el presidente y el secretario de esa reunión como prueba de ello.
+
+
```
Comment (bvhme) [inline organization/articles-of-association.es.md:163]: Multiple blank lines.

```suggestion

```

File: organization/articles-of-association.es.md [hunk 2]
```diff
@@ -0,0 +1,459 @@
...
+13.1. La junta directiva está a cargo de la gestión de la asociación.
+
+13.2. Cada director está obligado a ejercer debidamente sus funciones dentro de la asociación. Cada director es responsable del curso general de la asociación.
+
+13.3. La junta estará autorizada a decidir la celebración de acuerdos para la adquisición, enajenación y gravamen de bienes registrados y a celebrar acuerdos en los que la asociación se comprometa como garante o codeudor solidario, garantice a un tercero o se vincule como garantía de una deuda de otro siempre en concordancia con el plan de gestión política de la asociación.
+
+13.4. En todos los casos en que la asociación tenga un conflicto de intereses con uno o más directores, la Junta General podrá designar a una o más personas para que la representen. Un director estará obligado a informar a la asamblea general y a sus codirectores de un conflicto de intereses.
+
+
```
Comment (bvhme) [inline organization/articles-of-association.es.md:198]: Multiple blank lines

```suggestion

```

File: organization/articles-of-association.es.md [hunk 3]
```diff
@@ -0,0 +1,455 @@
+> El presente documento contienen una traducción no oficial desarrollada únicamente con fines informativos. El texto vinculante, por tanto, será el del [documento holandés original](https://github.com/publiccodenet/about/blob/develop/organization/articles-of-association.nl.md) en todos los aspectos legales.
+
+# Artículos de la asociación 'Foundation for Public Code Vereniging'
+
+El 3 de junio de dos mil diecinueve, el Sr. Lourens Wouter de Waard, notario con sede en Ámsterdam, comparece ante mí:
+
+Doña Ellen Tine Vermeulen, abogada, empleada en las oficinas de la empresa privada de responsabilidad limitada: Arcagna B.V., que tiene su domicilio social en Ámsterdam y con dirección: 1071 DJ Amsterdam, Museumplein 5 E&F, nacida en Ede el veintidós de agosto de mil novecientos setenta y dos, actuando en esta capacidad como apoderada escrita de:
+
+Boris Hendrik van Hoytema (datos eliminados deliberadamente) (el "Fundador 1");
```
Comment (ElenaFdR) [inline organization/articles-of-association.es.md:9]: This is a numbered list in the other versions.

**Comment by AlbaRoza** [comment]
Modified markdown. Travis still giving me problems.

File: organization/articles-of-association.es.md (hunk 3 above)
Comment (AlbaRoza) [reply organization/articles-of-association.es.md]: Done! 👍

**Comment by bvhme** [comment]
Hey @AlbaRoza 

Travis reports 1 problem with the file:

> `organization/articles-of-association.es.md:105: MD009 Trailing spaces`

This means that on line 105 there is a space at the end of the line, remove that and it should pass.

**Comment by bvhme** [comment]
To answer your question Alba, you have not removed the spaces. I'll do it and merge them and perhaps in the near future we can do a session to help you contribute.

**Comment by bvhme** [comment]
Sorry for my tone! I'm just a bit anxious that we have this great content and have not integrated it yet, which doesn't serve it right!

I've added the documents to the index and fixed the syntax. It should be good to go now. What do you say @ElenaFdR?

**Comment by ElenaFdR** [comment]
It looks great! Hurrah for sharing this more wildly :)

**Review by ElenaFdR — APPROVED** [review]

**Comment by ElenaFdR** [comment]
> It looks great! Hurrah for sharing this more wildly :)

🤔 I think I meant 'widely', but we can do both.

**Review by bvhme — APPROVED** [review]

**Pull request merged**